lib/astra-notices/class-astra-notices.php

```diff
@@ -28,7 +28,7 @@ class Astra_Notices {
 		 * @var array Notices.
 		 * @since 1.0.0
 		 */
-		private static $version = '1.1.8';
+		private static $version = '1.1.11';
```

Comment:
revert changes in the lib folder entirely as we do not usually scan it with.

Reply:
reverted changes